src/ui/InputPrepTime.tsx

```diff
@@ -0,0 +1,45 @@
+import { Field } from 'formik';
+import styled from '@emotion/styled';
+import { FC } from 'react';
+
+const StyledField = styled(Field)`
+  background: var(--input-background);
+  border-bottom: 1px var(--text) solid;
+  color: var(--text);
+  height: 51px;
+  padding-left: 10px;
+  margin-bottom: 10px;
+  font-family: monospace;
+`;
+
+interface InputPrepTimeProps {
+  name: string;
+}
+
+const InputPrepTime: FC<InputPrepTimeProps> = (name) => {
+  return (
+    <div>
```

Comment:
Let's start using the <Flex /> component when you just need a div wrapper. Altho we will have to rewrite the Flex component a little bit bc i stole it from work.